translations/texts/objects/apex/apexbed/apexbed.object.json

```
@@ -20,7 +20,7 @@
     },
     "Texts": {
       "Eng": "A simple bed, lacks any real charm.",
-      "Rus": ""
+      "Rus": "Простая кровать, не хватает настоящего шарма."
```

```suggestion
      "Rus": "Простая кровать. Ей явно не хватает хоть сколько-нибудь очарования."
```
Real перевели, а про any забыли. Речь идёт не про то, что ей не хватает именно настоящего шарма, а про то, что она лишена вообще хоть какого.